.github/workflows/branch-fork.yml

```diff
@@ -0,0 +1,48 @@
+name: Fork Branch
+
+on:
+  push:
+    branches:
+      - "**"
+
+defaults:
+  run:
+    working-directory: src/client
+
+jobs:
+  build:
+    name: Build
+
+    runs-on: ubuntu-20.04
+
+    if: github.event.repository.full_name != github.repository
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set Node to v16
```

Comment:
Minor, Node 20 now

Reply:
ah, the comment, yes .. I knew there would be something
as Node 20 seems to be all good to go, we should bump to this locally and test it all out then update the `engine` in package.json too